pandas/core/generic.py

```diff
@@ -11895,7 +11895,7 @@ def _logical_func(
 
     def any(
         self,
-        axis: Axis = 0,
+        axis: Axis | None = 0,
```

Comment:
unrelated?

Reply:
mypy discovered this via my use of `.any(None)` in `_idxmax_idxmin` here.